.github/workflows/test.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
             - uses: shivammathur/setup-php@v2
               with:
                   php-version: '7.4'
-                  extensions: gd
+                  extensions: gd imagick
```

Comment:
It's a CSV list https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support

Reply:
Duh, sorry. I corrected that, and added it in the other places it's set too.